content/docs/concepts/traffic-management/index.md

```diff
@@ -513,6 +513,7 @@ spec:
     retries:
       attempts: 3
       perTryTimeout: 2s
+      retryOn: gateway-error,connect-failure,refused-stream
```

Comment:
What's the default value for retryOn. If it is reasonable, it might be better to leave this field out of the simple example in this concepts section. If it is needed, we should probably at least mention it in the text above the example too.

Reply:
No 5xx isn't in the default  values list of `retryOn`.